index.html

```diff
@@ -339,12 +339,9 @@ <h2 class="contact-title">Contact me</h2>
               love to help with it
             </p>
           </div>
-          <form novalidate
-            action="https://formspree.io/f/xeqvdqoj"
-            method="post"
-            class="contact-form"
-            id="contact-form"
-          >
+          action="https://formspree.io/f/xeqvdqoj" method="post"
+          class="contact-form" id="contact-form" >
+          <form>
```

Comment:
- [OPTIONAL] It would be better if you remove this text from your contact form. ♻️ 
- [OPTIONAL] Great piece of code to populate your popup window using the information of your array of objects. However there is a part that is missing, You should create your project cards using that information too. It means that already created HTML code for the projects list should be removed and instead it should be added via JavaScript. It will allow you to create projects cards dynamically. Don't forget that your function for creating those cards in JavaScript needs to run when the page loads. 😉(It is a requirement so please do it).